src/main/java/redis/clients/jedis/JedisPool.java

```diff
@@ -145,6 +155,11 @@ public JedisPool(final GenericObjectPoolConfig poolConfig, final String host, in
     this(poolConfig, host, port, timeout, timeout, password, database, clientName);
   }
 
+  public JedisPool(final GenericObjectPoolConfig poolConfig, final String host, int port,
+                   int timeout, final String user, final String password, final int database, final String clientName) {
+    this(poolConfig, host, port, timeout, timeout, password, database, clientName);
+  }
+
```

Comment:
Add at least one more constructor for ssl.

https://github.com/xetorthio/jedis/blob/d1224ee03ca34c80e013141ea9990db40ebd2fc9/src/main/java/redis/clients/jedis/JedisPool.java#L177-L180

Reply:
While doing this I realize that I introduce a regression in SSLJedisClusterTest, fixing it.

Reply:
I have implemented multiple constructors, and also added more tests around SSL and Cluster